engine/reference/commandline/dockerd.md

```diff
@@ -1272,7 +1272,7 @@ pid file here.
 - `--iptables=false` prevents the Docker daemon from adding iptables rules. If
   multiple daemons manage iptables rules, they may overwrite rules set by
   another daemon. Be aware that disabling this option requires you to manually
-  add iptables rules to expose container ports.
+  add iptables rules to expose container ports. Disabling this also avoids creating nat masquerade rules even if --ip-masq is set to true. This will result in no external connectivity outside of the docker host when using networks other than the default bridge.
```

Comment:
nit: Please wrap long lines at 80 columns if possible.

Consider simplifying the wording, to something like:

If you prevent Docker from adding `iptables` rules, Docker will also not add IP masquerading rulesr, even if you set `--ip-masq` to `true`. Without IP masquerading rules, no external host will be able to connect to the containers managed by this Docker daemon and Docker containers will not be able to connect to external hosts or the internet.

(I think the last part is true, am I correct?)